crates/nu-utils/src/default_files/sample_config.nu

```diff
@@ -480,27 +487,22 @@ $env.config.menus ++= [
 # Plugin behavior
 # ---------------
 # Per-plugin configuration. See https://www.nushell.sh/contributor-book/plugins.html#configuration.
-plugins: {}
 $env.config.plugins
+
+# Configuration for plugin garbage collection
 $env.config.plugin_gc
 $env.config.plugin_gc.default
+# true to enable stopping of inactive plugins
```

Comment:
```suggestion
# true to enable stopping inactive plugins
```

Reply:
I know this was the original text that I left in from the original config - Just noticing it could be improved as long as it's getting moved :-)